protocols/kad/src/behaviour/test.rs

```diff
@@ -71,7 +72,8 @@ fn build_node_with_config(cfg: Config) -> (Multiaddr, TestSwarm) {
         transport,
         behaviour,
         local_id,
-        swarm::Config::with_async_std_executor(),
+        swarm::Config::with_async_std_executor()
+            .with_idle_connection_timeout(Duration::from_secs(60 * 5)),
```

Comment:
I am surprised that we need this. I would have thought that the tests might pass without it. We should set it to something way shorter though.

```suggestion
            .with_idle_connection_timeout(Duration::from_secs(5)),
```

Reply:
fixed 78ebe38